src/Console/Commands/GenerateTypesCommand.php

```diff
@@ -96,31 +107,40 @@ public function __construct()
     /**
      * Execute the console command.
      *
+     * @throws Exception
      * @return void
-     * @throws \Exception
      */
     public function handle(): void
     {
         $this->modelDir = $this->option('modelDir') ?? config('laravel-model-ts-type.model_dir');
         $this->namespace = $this->option('namespace') ?? config('laravel-model-ts-type.namespace');
         $this->outputDir = $this->option('outputDir') ?? config('laravel-model-ts-type.output_dir');
         $this->useKebabCase = !($this->option('noKebabCase') ?? config('laravel-model-ts-type.no_kebab_case'));
+        $this->model = $this->option('model') ?? null;
+
         $this->getModels($this->modelDir);
 
         foreach ($this->modelHits as $model) {
-            $fullyQualifiedName = format_namespace($model);
+            $fullyQualifiedName = FormatNamespace::get($model);
 
             if (!in_array($fullyQualifiedName, get_declared_classes())) {
-                include($model);
+                include_once $model;
             }
 
             $reflectionClass = new \ReflectionClass($fullyQualifiedName);
 
-            if ($reflectionClass->isAbstract()) {
+            if ($reflectionClass->isAbstract()
+                || $reflectionClass->isTrait()) {
```

Comment:
this can go on 1 line

Reply:
Switch the `isAbstract()` with `isTrait()` or? Because the ReflectionClass is init'd right above this check.